docs/sources/enterprise/kms-integration/using-google-cloud-kms-to-encrypt-database-secrets.md

```diff
@@ -0,0 +1,70 @@
++++
+title = "Google Cloud KMS"
+description = "Using Google Cloud KMS to encrypt database secrets"
+keywords = ["grafana", "Google Cloud KMS integration"]
+weight = 3
++++
+
+# Using Google Cloud KMS to encrypt database secrets
+
+You can use an encryption key from Google Cloud Key Management Service to encrypt secrets in the Grafana database.
+
+**Prerequisites:**
+
+- A Google Cloud account with permission to list and create KMS keys and service accounts to access those keys
+- Access to the Grafana [configuration]({{< relref "../../administration/configuration/#config-file-locations" >}}) file
+
+1. [Create a key ring](https://cloud.google.com/kms/docs/creating-keys#kms-create-key-ring-console).
+
+2. [Create a symmetric encryption key](https://cloud.google.com/kms/docs/creating-keys#create_a_key) in the key ring on the specified location.
+
+3. [Create a service account](https://cloud.google.com/iam/docs/creating-managing-service-accounts#creating) and assign it a role: it can be a predefined role or custom role with permissions to encrypt and decrypt secrets with Key Management Service.
+
+4. [Create and download a JSON file containing service account credentials](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating).
+
+5. From within Grafana, turn on [envelope encryption]({{< relref "../../administration/envelope-encryption.md" >}}).
+
+6. Add your Google Cloud KMS details to the Grafana configuration file; depending on your operating system, is usually named `grafana.ini`:
+   <br><br>a. Add a new section to the configuration file, with a name in the format of `[security.encryption.azurekv.<KEY-NAME>]`, where `<KEY-NAME>` is any name that uniquely identifies this key among other provider keys.
+   <br><br>b. Fill in the section with the following values:
+   <br>
+
+   - `key_id`: encryption key ID, refer to [Getting the ID for a Key](https://cloud.google.com/kms/docs/getting-resource-ids#getting_the_id_for_a_key_and_version).
+   - `credentials_file`: path to the JSON file containing a service account credentials.
```

Comment:
Is this path relative to the config file itself?

Reply:
Clarified.